packages/acs-ui-common/CHANGELOG.md

```diff
@@ -1,94 +0,0 @@
-# Change Log - @internal/acs-ui-common
```

Comment:
Should the `changelog.json` also be deleted per packlet? (and just have one under `communication-react`)

Reply:
Can't quite delete it right now. As long as there are changefiles for the packlets, collecting the changelogs will update these JSON files.

Once we're done with the existing changefiles for these packlets (i.e., once these PRs land and we've released a beta and a stable release), we can consider deleting the various CHANLOG.json files.